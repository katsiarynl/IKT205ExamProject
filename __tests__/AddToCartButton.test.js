import React from "react";
import { create } from "react-test-renderer";
import AddToCartButtonComponent from "../components/products/AddToCartButton";
const tree = create(<AddToCartButtonComponent />);

test("snapshot", () => {
  expect(tree).toMatchSnapshot();
});