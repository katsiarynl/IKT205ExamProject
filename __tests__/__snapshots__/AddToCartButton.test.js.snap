// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={false}
  >
    <View
      style={
        {
          "alignItems": "center",
          "backgroundColor": "orange",
          "borderRadius": 15,
          "height": 66.7,
          "justifyContent": "center",
          "width": 187.5,
        }
      }
    >
      <Text
        style={
          {
            "fontSize": 18,
            "fontWeight": "bold",
          }
        }
      >
        Add To Cart
      </Text>
    </View>
  </View>
</View>
`;
